src/components/decoration/textBlocks.tsx

```diff
@@ -60,11 +62,12 @@ export const SectionHeader = ({
     .filter(Boolean)
     .filter((component) => component !== boldStart && component !== boldEnd);
 
-  const boldClasses =
-    'text-5xl md:text-6xl font-mono font-semibold uppercase mx-1';
+  const boldClasses = `${
+    smallOnMobile ? 'text-4xl' : 'text-5xl'
+  } md:text-6xl font-mono font-semibold uppercase mx-1`;
   const italicClasses = classNames(
-    'font-serif italic text-4xl mx-1',
-    'font-bold',
+    `${smallOnMobile ? 'text-3xl md:' : ''}text-4xl font-serif italic  mx-1',
+    'font-bold`,
```

Comment:
Propably should be
 ```
`${smallOnMobile ? 'text-3xl md:' : ''}text-4xl font-serif italic  mx-1`,
    'font-bold'
```

Reply:
What's the difference sorry?

Reply:
Small but important difference in quotes :)
`${smallOnMobile ? 'text-3xl md:' : ''}text-4xl font-serif italic  mx-1',
    'font-bold`
`${smallOnMobile ? 'text-3xl md:' : ''}text-4xl font-serif italic  mx-1`,
   'font-bold'